index.html

```diff
@@ -12,178 +12,26 @@ <h1>SlideShow</h1>
-			</div>
-			<div class="slideshow__slide">
-				<div class="slideshow__title" onclick="showReference()">
-					<h3>Lutry, Switzerland</h3>
-				</div>
-				<img src="https://images.unsplash.com/photo-1517483141275-296bb15890b2?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=f6fb898eac03ab9abd1137fa691ebe82&auto=format&fit=crop&w=1051&q=80" alt="Lutry, Switzerland by Olivier Fahrni">
-				<div class="slideshow__paragraph">
-					<p>Lutry is a municipality in the Swiss canton of Vaud, located in the Lavaux-Oron.</p>
-				</div>
-				<div class="slideshow__reference">Lutry, Switzerland by Olivier Fahrni</div>
-			</div>
-			<div class="slideshow__slide">
-				<div class="slideshow__title" onclick="showReference()">
-					<h3>Mirror Lake Trail - Tosemite National Park</h3>
-				</div>
-				<img src="https://images.unsplash.com/photo-1480499484268-a85a2414da81?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=6d765c5039bd6d36022739e3a6552c44&auto=format&fit=crop&w=1055&q=80" alt="Mirror Lake, Yosemite Valley, United States by Quentin Dr">
-				<div class="slideshow__paragraph">
-					<p>From the shuttle stop (#17), a paved trail leads directly to Mirror Lake. Which is in the Adirondack Mountains in northern New York in the United States. The lake is approximately 124 acres, with a watershed area of 741 acres.</p>
-				</div>
-				<div class="slideshow__reference">Mirror Lake, Yosemite Valley, United States by Quentin Dr</div>
-			</div>
-			<div class="slideshow__thumbnails"> 
-				<div class="slideshow__thumbnail-imgs slideshow__thumbnail-imgs--actual" onclick="showSlide(idx, idx = 1)"><img src="https://images.unsplash.com/photo-1523977843538-0650e8cb2e7d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=a43d0d3984b7a6e57716c15ee6fe5797&auto=format&fit=crop&w=1050&q=80"></div> 
-				<div class="slideshow__thumbnail-imgs" onclick="showSlide(idx, idx = 2)"><img src="https://images.unsplash.com/photo-1491904768633-2b7e3e7fede5?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=b201ea4f1683a8e092ff153c1b423e78&auto=format&fit=crop&w=1189&q=80"></div>
-				<div class="slideshow__thumbnail-imgs" onclick="showSlide(idx, idx = 3)"><img src="https://images.unsplash.com/photo-1516524609276-3cb95aa22190?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=3d79a0b6dc9683effb116d7e9763bfa2&auto=format&fit=crop&w=1225&q=80"></div>
-				<div class="slideshow__thumbnail-imgs" onclick="showSlide(idx, idx = 4)"><img src="https://images.unsplash.com/photo-1517483141275-296bb15890b2?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=f6fb898eac03ab9abd1137fa691ebe82&auto=format&fit=crop&w=1051&q=80"></div>
-				<div class="slideshow__thumbnail-imgs" onclick="showSlide(idx, idx = 5)"><img src="https://images.unsplash.com/photo-1480499484268-a85a2414da81?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=6d765c5039bd6d36022739e3a6552c44&auto=format&fit=crop&w=1055&q=80"></div>
+				<div class="slideshow__reference" style="display: none;">Fort Ebey State Park, Coupeville, United States by Patrick Fore.</div>
```

Comment:
why prefer having `style="display:none" ` inline and not in css ?

Reply:
I am not sure myself, but it is for the toggle of display='none' or display='block'

Reply:
jummm I think is better to handle this using classes. e.g. : `slideshow__paragraph` `slideshow__paragraph hidden`